docs/source/quick_reference.rst

```diff
@@ -169,6 +176,14 @@ Stroke Name
 An open "tag" to identify an instrument stroke name or type. Used for instruments that have specific
 stroke labels.
 
+.. _syllables:
+
+Syllables
+^^^^^^^^^
+Additional representation of the sung lyrics but structured as syllables instead of complete sentences. It can
+be annotated as time-aligned events where the events are the syllables happening at certain time
+intervals. Otherwise, they can be represented as a stream of strings, grouped by meaningful syllable structures.
+
 .. _tonic:
```

Comment:
@nkundiushuti why are you referring to stroke-name, tempo and technique?

Reply:
@nkundiushuti, @magdalenafuentes this dataset has `Lyrics`, `LyricsChar` (which are Lyrics written in Chinese), `Syllables`, and `Phonemes`. Syllabes and Phonemes were not represented in `quick_reference.rst` yet, so I added both here. The annotations for the whole set of datasets are here, so that's why stroke-name, tempo, etc. are present in `quick_reference.rst` but they are not annotations of the Jingju Acappella Dataset. Is that what you mean?